File: servico-musica-java/java.graphql/src/main/java/servico/java/graphql/controller/MusicaController.java
package servico.java.graphql.controller;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.graphql.data.method.annotation.QueryMapping;
import org.springframework.stereotype.Controller;
import servico.java.graphql.entity.Musica;
import servico.java.graphql.repository.MusicaRepository;

import java.util.List;

@Controller
public class MusicaController {

    @Autowired
    private MusicaRepository musicaRepository;

    @QueryMapping
    public List<Musica> todasAsMusicas() {
        return musicaRepository.findAll();
    }
}

File: servico-musica-kotlin/kotlin.graphql/src/main/kotlin/servico/kotlin/graphql/controller/MusicaController.kt
package servico.kotlin.graphql.controller

import org.springframework.graphql.data.method.annotation.QueryMapping
import org.springframework.stereotype.Controller
import servico.kotlin.graphql.entity.Musica
import servico.kotlin.graphql.repository.MusicaRepository

@Controller
class MusicaController (
    private val musicaRepository: MusicaRepository
) {

    @QueryMapping
    fun todasAsMusicas(): List<Musica> {
        return musicaRepository.findAll()
    }
}